src/content/docs/highlights/v0.26.3.md

```diff
@@ -0,0 +1,190 @@
+enthusiasts.
+
+![ratatui-forum](https://github.com/ratatui-org/ratatui/assets/24392180/b13016ca-1044-4c34-9687-f0e39167fb14)
+
+<center>
+
+Join here: [**https://forum.ratatui.rs**](https://forum.ratatui.rs)
+
+</center>
+
+Here you can get help with your Rust/Ratatui questions and share your projects!
+
+---
+
+## Fix Unicode Truncation Bug 🐛
+
+If you are using Ratatui `0.26.2` you might have hit this bug:
+
+> panic occurred at `ratatui-0.26.2/src/text/line.rs:477:59` byte index 51 is not a char boundary;
+> it is inside 'で' (bytes 49..52) of
+> `🦀 RFC8628 OAuth 2.0 Device Authorization GrantでCLIからGithubのaccess tokenを取得する`
+
+This issue was introduced in [this PR](https://github.com/ratatui-org/ratatui/pull/987) and now
+fixed with `0.26.3`!
+
+```rs
+#[test]
+fn truncation_works_with_emoji() {
+    let line = Line::raw( "123456789🦀");
+    let mut buf = Buffer::empty(Rect::new(0, 0, 10, 1));
+    line.render(buf.area, &mut buf);
+    assert_buffer_eq!(buf, Buffer::with_lines(vec!["123456789 "]));
+}
+```
+
+- Details: <https://github.com/ratatui-org/ratatui/issues/1032>
+- Implementation: <https://github.com/ratatui-org/ratatui/pull/1089>
+
+---
+
+## Color: Better Serialization 🎨
+
+`Color::Rgb` will now be serialized as the hex representation of their value.
+
+For example, `Color::Rgb(255, 0, 255)` would be serialized as `"#FF00FF"` rather than
+`{"Rgb": [255, 0, 255]}`:
+
+```rs
+let json_rgb = serde_json::to_string(&Color::Rgb(255, 0, 255))?;
+assert_eq!(json_rgb, r##""#FF00FF""##);
+assert_eq!(
+    serde_json::from_str::<Color>(&json_rgb)?,
+    Color::Rgb(255, 0, 255)
+);
+```
+
+Similarly, `Color::Indexed` will now be serialized as just the string of the index.
+
+For example, with serde_json, `Color::Indexed(10)` would be serialized as `"10"` rather than
+`{"Indexed": 10}`:
+
+```rs
+let json_indexed = serde_json::to_string(&Color::Indexed(10))?;
+assert_eq!(json_indexed, r#""10""#);
+assert_eq!(
+    serde_json::from_str::<Color>(&json_indexed)?,
+    Color::Indexed(10)
+);
+```
+
+---
+
+## Faster Rendering 🚀
+
+We sped up combined foreground and background color changes for the `crossterm` backend by up to
+20%! 🔥
+
+For more information, see:
+
+- <https://github.com/ratatui-org/ratatui/pull/1072>
+- <https://github.com/crossterm-rs/crossterm/pull/879>
+
+> I changed the SetColors command to write both colors at once with a single write instead of
+> multiple writes that more bytes. This led to a 15-25% fps increase when testing the colors_rgb
+> example on iTerm2 on an M2 Macbook Pro.
+
+---
+
+## Deprecate `assert_buffer_eq` macro 🚫
+
+[`assert_buffer_eq`](https://docs.rs/ratatui/latest/ratatui/macro.assert_buffer_eq.html) is now
+deprecated in favor of `assert_eq`:
+
+```diff
+-assert_buffer_eq!(actual, expected);
++assert_eq!(actual, expected);
+```
+
+We also introduced a new macro called `assert_buffer_lines` for checking if `TestBackend`'s buffer
+is equal to the expected lines.
```

Comment:
It's a new function on `TestBackend`. `TestBackend::assert_buffer_lines`. It's to simplify `TestBackend::assert_buffer(&Buffer::with_lines(…))`. The current wording sounds like it's a replacement for any `assert_eq!()` with buffers.

Reply:
Feel like slapping a quick PR together for this so this comment doesn't get lost?

Reply:
https://github.com/ratatui-org/ratatui-website/pull/612